documentation/docs/25-build-and-deploy/70-adapter-cloudflare-workers.md

```diff
@@ -77,6 +77,22 @@ export default {
 };
 ```
 
+If you would like to enable [Node.js compatibility](https://developers.cloudflare.com/workers/runtime-apis/nodejs/#enable-nodejs-from-the-cloudflare-dashboard), you can do like so:
+
+```js
+// @errors: 2307
+/// file: svelte.config.js
+import adapter from '@sveltejs/adapter-cloudflare-workers';
+
+export default {
+	kit: {
+		adapter: adapter({ nodeCompat: true })
```

Comment:
why pass it in if it's already specified in the `wrangler.toml`? https://developers.cloudflare.com/workers/runtime-apis/nodejs/#enable-nodejs-with-workers

I think it'd be nicer to read it from the `wrangler.toml` to avoid making the user specify it twice

Reply:
When deploy to Cloudflare Pages, the file `wrangler.toml' is not available.

Reply:
Yes, but this is the Cloudflare Workers adapter :grin:

Reply:
Oke, I'm going to make it read the `wrangler.toml`, not sure how could I do that. Maybe just `string.includes('node_compat')`? 🤔

Reply:
oh, there is a `toml.parse` already.